src/components/heading/style.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledHeading = styled.h1`
+  color: ${(props) => props.color || '#222'};
+`;
+
+const Heading = ({ level, ...rest }) => (
+  <StyledHeading as={`h${level}`} {...rest} />
+);
+
+Heading.defaultProps = {
+  level: 1,
+};
+
+export default Heading;
```

Comment:
This component isn't necessary. GlobalStyles will cover this styling. Just use normal H tags.